Jint/Native/RegExp/RegExpConstructor.cs

```diff
@@ -137,40 +137,13 @@ private ObjectInstance RegExpInitialize(RegExpInstance r, JsValue pattern, JsVal
             return r;
         }
 
-        private RegExpInstance RegExpAlloc()
+        private RegExpInstance RegExpAlloc(JsValue newTarget)
         {
-            var r = new RegExpInstance(Engine)
-            {
-                _prototype = PrototypeObject
-            };
-            return r;
-        }
-
-        public RegExpInstance Construct(string regExp, Engine engine)
-        {
-            var r = new RegExpInstance(Engine);
-            r._prototype = PrototypeObject;
-
-            var scanner = new Scanner(regExp, new ParserOptions { AdaptRegexp = true });
```

Comment:
Where did this go?

Reply:
It was never used, so removed dead code with extreme prejudice.